en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
-- Choose a frame - [Airframe Builds](airframes/index.md) lists the supported frames and provides detailed instructions for how to construct a subset of vehicles.
-- Choose a flight controller - see [Getting Started > Flight Controllers](getting_started/flight_controller_selection.md) and [Autopilot Hardware](flight_controller/index.md).
-- [Assembly](assembly/index.md) explains how to wire up the important peripherals to your autopilot.
-- [Basic Configuration](config/index.md) shows how to update your firmware and configure it with settings appropriate for your airframe.
-  This section also explains how to calibrate the main sensors (compass, gyro/IMU, airspeed etc.), and setup your remote control and safety features.
+Before you fly you should read [Operations](config/operations.md) to understand how to set up the safety features of your vehicle and the common behaviours of all frame types.
+Once you've done that you're ready to fly.
 
-Once you are ready to fly your vehicle, visit the [Flying](flying/index.md) section.
+Basic instructions for flying each vehicle type are provided in the respective sections, such as [Basic Flying (Multicopter)](flying/basic_flying_mc.md).
 
-**I want to add payload or a camera:**
+### I want to add payload or a camera
 
-The payloads section describes how to add a camera or how to configure PX4 to enable you to deliver packages.
+The [Payloads](payloads/index.md) section describes how to add a camera or how to configure PX4 to enable you to deliver packages.
 
-- [Payloads](payloads/index.md) describes how to integrate payloads
+After adding a payload you should retune the vehicle, which for multicopters you can do with [Autotune](config/autotune.md).
+You should also consider the other tuning and calibration listed in your vehicle configuration guide (e.g. [Multicopter Configuration/Tuning](config_mc/index.md)).
 
-**I am modifying a supported vehicle:**
+### I am modifying a supported vehicle
 
-Modifications of the flight controller and basic sensors are covered above.
-In order to use new sensors, or if you have made changes that significantly affect flight characteristics:
+The [Hardware Selection & Setup](hardware/drone_parts.md) section provides both high level and product-specific information about hardware that you might use with PX4 and its configuration.
+This is the first place you should look if you want to modify a drone and add new components.
 
-- [Peripheral Hardware](peripherals/index.md) provides additional information about using external sensors.
-- [Basic Configuration](config/index.md) explains how to calibrate the main sensors.
-- [Advanced Configuration](advanced_config/index.md) should be used to re/fine-tune the airframe.
+Just as when you add a payload, you should re-tune the vehicle (see [Multicopter Configuration/Tuning](config_mc/index.md) for links to the relevant information).

```

Comment:
I really wouldn't stress this so much. Probably it's rather a detail that needs to be in the steps of adding a payload e.g. how to mount it, does it have a big influence on the weight, airflow, distance or airspeed sensor?

Also I'd point to one tuning landing page and then explain there autotuning, manual tuning and a procedure to try them out.

Reply:
OK, so we'll have to add those notes to the payload page. I think I'll just remove this here. We're not trying to explain everything, just provide board pointers.

```suggestion

```